package.json

```diff
@@ -50,6 +50,7 @@
     "lib-jitsi-meet": "github:jitsi/lib-jitsi-meet#05b7ae60dc669a69b293fb0e93688fc0a78adb80",
     "lodash": "4.17.4",
     "moment": "2.19.4",
+    "moment-duration-format": "^2.2.2",
```

Comment:
Notice that we are using the exact versions for our dependancies. Please use the exact version here as well.